README.md

```diff
@@ -46,71 +52,17 @@ submitting Pull Requests.
-* [@karianna](https://github.com/karianna) - Martijn Verburg (Microsoft / LJC) - *
-* [@mwornast](https://github.com/mwornast) - Marcus Wornast (IBM) - *
-* [@pnstanton](https://github.com/pnstanton) - Peter Stanton (IBM) - *
-* [@sej-jackson](https://github.com/sej-jackson) - Sej Jackson (IBM)
-* [@sxa555](https://github.com/sxa555) - Stewart X Addison (IBM) - *
-* [@tellison](https://github.com/tellison) - Tim Ellison (Red Hat) - *
-* [@vsebe](https://github.com/vsebe) - Violeta Sebe (IBM)
-* [@Willsparker](https://github.com/Willsparker) - William Parker (AdoptOpenJDK) - *
-
-## [@adoptopenjdk-infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/adoptopenjdk-infrastructure)
-
-Issues can be assigned to these folks
-
-* [@AdamBrousseau](https://github.com/AdamBrousseau) - Adam Brousseau (IBM)
-* [@CJKwork](https://github.com/CJKwork) - Clive Kennedy (IBM)
-* [@cwesMills](https://github.com/cwesMills) - Colton Mills (IBM)
-* [@cwillhelm](https://github.com/cwillhelm) - Connor Willhelm (IBM)
-* [@Haroon-Khel](https://github.com/Haroon-Khel) - Haroon Khel (Red Hat)
-* [@HusainYusafali](https://github.com/HusainYusufali) - Husain Yusufali (IBM)
-* [@jdekonin](https://github.com/jdekonin) - Joe deKoning (IBM)
-* [@karianna](https://github.com/karianna) - Martijn Verburg (Microsoft / LJC)
-* [@sej-jackson](https://github.com/sej-jackson) - Sej Jackson (IBM)
-* [@vsebe](https://github.com/vsebe) - Violeta Sebe (IBM)
-* [@Willsparker](https://github.com/Willsparker) - William Parker (AdoptOpenJDK)
-
-### [@jenkins-admins](https://github.com/orgs/AdoptOpenJDK/teams/jenkins-admins)
-
-* [@ali-ince](https://github.com/ali-ince) Ali Ince (LJC)
-* [@andrew-m-leonard](https://github.com/andrew-m-leonard) Andrew M Leonard (Red Hat)
-* [@gdams](https://github.com/gdams) - George Adams (Microsoft)
-* [@geraintwjones](https://github.com/geraintwjones) - Geraint Jones (IBM)
-* [@johnoliver](https://github.com/johnoliver) - John Oliver (Microsoft / LJC)
-* [@karianna](https://github.com/karianna) - Martijn Verburg (Microsoft / LJC)
-* [@mwornast](https://github.com/mwornast) - Marcus Wornast (IBM)
-* [@neomatrix369](https://github.com/neomatrix369) - Mani Sarkar (LJC)
-* [@smlambert](https://github.com/smlambert) - Shelley Lambert (Red Hat)
-* [@sxa555](https://github.com/sxa555) - Stewart X Addison (Red Hat)
-* [@tellison](https://github.com/tellison) - Tim Ellison (Red Hat)
-* [@VermaSh](https://github.com/VermaSh) Shubham Verma (IBM)
+* [infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/infrastructure) - Can manage issues and PRs at GitHub.
+* [infrastructure-core](https://github.com/orgs/AdoptOpenJDK/teams/infrastructure-core) - admin access to openjdk-infrastructure.
+* [infrastructure-secret](https://github.com/orgs/AdoptOpenJDK/teams/infrastructure-secret) - higher level of access for system administrators only.
+* [infrastructure-triage](https://github.com/orgs/AdoptOpenJDK/teams/infrastructure-triage) - starting team for new joiners.
```

Comment:
Might be worth leaving the "full access" admin team in this document so that people know who is able to make full admin level decisions if required, but I agree that leaving the other lists in here is excessive from a maintenance perspective. I'd also be tempted to move point people at the ONBOARDING.md doc for the access list, since we seem to have it duplicated in multiple places at the moment.

The description of the `infrastructure-core` should probably be adjusted in line with https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1084#issuecomment-688443553 to be "The team who can administer machines" (in jenkins and those who have admin access to the machines directly)